src/Compilers/CSharp/Portable/Symbols/Source/ModifierUtils.cs

```diff
@@ -44,7 +45,8 @@ internal static class ModifierUtils
                 switch (oneError)
                 {
                     case DeclarationModifiers.Partial:
-                        diagnostics.Add(ErrorCode.ERR_PartialMethodOnlyMethods, errorLocation);
+                        // Errors about 'partial' are reported in ToDeclarationModifiers.  So no need to report
```

Comment:
The comment is not quite right. There is one path that calls `CheckModifiers` directly. See `SourceMemberContainerSymbol.cs` `MakeAndCheckTypeModifiers`.